mod/wikirate/set_pattern/citable.rb

```diff
@@ -0,0 +1,17 @@
+
+
+def label name
+  'All Card with type "Web Source", "File Source" and "Text Source" cards'
+end
+
+def prototype_args anchor
+  { :name=>"*dummy", :type_id => Card::SourceFileID }
+end
+
+def pattern_applies? card
+  card.type_id == Card::SourceFileID || card.type_id == Card::WebpageID || card.type_id == Card::TextSourceID 
+end
+
+def follow_label name
+  label name
+end
```

Comment:
this should return something shorter so that it fits in the gear menu. I suggest "all citeable sources". It's also important that the first letter is lowercase because in the follow submenu the word "follow" is prepended